default/stringtables/en.txt

```diff
@@ -6283,6 +6295,21 @@ EFFECT_BIOWEAPON
 EFFECT_BIOWEAPON_LABEL
 Bioweapon Attack
 
+EFFECT_PLANET_TARGETED
+At %planet%: our planet is being targeted by a planet destroyer cannon from the ship %ship%!
+
+EFFECT_PLANET_DESTRUCTION
+We have destroyed the %empire% planet %planet%!
```

Comment:
Again, planet first, or possibly system (will the link still work after it's destroyed?)
At %system%: we destroyed %empire% %planet%

Reply:
It will point to the remaining asteroids.